server/build.go

```diff
@@ -495,15 +495,6 @@ func PostBuild(c *gin.Context) {
 		return
 	}
 
-	switch build.Status {
-	case model.StatusPending,
-		model.StatusRunning,
-		model.StatusDeclined,
-		model.StatusBlocked,
```

Comment:
I think we might want to keep `Declined` and `Blocked` since this typically indicates a build has a security-related change that needs manual evaluation. But otherwise, we should be able to remove all others.